hub/scripts.js

```diff
@@ -275,29 +277,32 @@
       const $up = $a.parentElement;
       const $twoup = $a.parentElement.parentElement;
       if ($up.childNodes.length === 1 && $up.tagName === 'P') {
-        $a.className = 'button secondary';
+        // Setting CTA parent element to `flex` in order to be able
+        // to apply all the suggested Spectrum styling.
+        // CTA buttons should not normally occur in the normal text,
+        // they should be present in the card component, so this
+        // is just a temporary solution for demo purposes
+        $up.style.display = 'flex';
+        $a.className = `${CONFIG.SELECTORS.CTA} ${CONFIG.SELECTORS.SECONDARY_CTA}`;
       }
       if ($up.childNodes.length === 1 && $up.tagName === 'STRONG'
         && $twoup.childNodes.length === 1 && $twoup.tagName === 'P') {
-        $a.className = 'button primary';
+        $twoup.style.display = 'flex';
+        $a.className = `${CONFIG.SELECTORS.CTA} ${CONFIG.SELECTORS.PRIMARY_CTA}`;
       }
     });
   }
 
-  function markReadyAfterDecorations() {
+  // Attach a 'ready' class to the main `div` once transformations are complete
+  function markPageAsReady() {
     const mainElement = document.querySelector(`${CONFIG.SELECTORS.MAIN}`);
```

Comment:
Why not just:
```suggestion
    const mainElement = document.querySelector(CONFIG.SELECTORS.MAIN);
```
A note aside: I know you like it organized, but `CONFIG.SELECTORS.MAIN` is 5x longer than `'main'` which will most likely never change...

Reply:
Sure, since it's a tag name, it makes sense to reference it directly. Hopefully the structure will not change, this is the main reason (pun intended) for which we like to keep such references grouped together.